docs/DeveloperGuide.adoc

```diff
@@ -785,6 +785,45 @@ Use case ends.
 +
 Use case resumes at step 2.
 
+*MSS*
+
+1.  User requests to show a list of person that is mostly accessed by keying in "list freq" keyword
+2.  AddressBook shows this list of person in descending frequency order
++
+Use case ends.
+
+*Extensions*
+
+[none]
+* 2a. The list is empty.
++
+Use case ends.
+
+*MSS*
+
+1.  User requests to show one of a person using find command
+2.  AddressBook shows this person
+3.  User requests to delete one of this person's tag by specifying a index of tag
```

Comment:
There might be something's wrong with the index. Besides, choosing tag based on index rather than its name seems inconvenient (my personal opinion)